models/UserModel.ts

```diff
@@ -0,0 +1,43 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+import { ApplicationStatus, UserAccessType } from '../types/Enums';
+
+@Entity()
+export class UserModel {
```

Comment:
Is there a reason you aren't doing the extends BaseEntity with the Model? How are you planning to handle create/merge functions without it?